src/rantanplan/core.py

```diff
@@ -63,23 +1336,55 @@ def have_prosodic_liaison(first_syllable, second_syllable):
 """
 
 
-def hyphenate(word):
+def apply_exception_rules(word):
+    # 'Abrogar' and derivatives
+    if ABROG_RE.match(word):
+        match = ABROG_RE.search(word)
+        if match is not None:
+            word = match.group(1) + '-' + match.group(2)
+    # 'Obrepticio' and derivatives
+    if OBREP_RE.match(word):
+        match = OBREP_RE.search(word)
+        if match is not None:
+            word = match.group(1) + '-' + match.group(2)
+    # Prefix 'en' followed by consonant
+    if PREFIX_EN_WITH_CONSONANT_RE.match(word):
+        match = PREFIX_EN_WITH_CONSONANT_RE.search(word)
+        if match is not None:
+            word = match.group(1) + '-' + match.group(2)
+    # Prefix 'des'/'sin' followed by consonant
+    if PREFIXES_WITH_CONSONANT_RE.match(word):
+        match = PREFIXES_WITH_CONSONANT_RE.search(word)
+        if match is not None:
+            word = match.group(1) + '-' + match.group(2)
+    # Group rh followed by u + dipthong
+    if PREFIX_RNH_DIPTHONG_RE.match(word):
+        match = PREFIX_RNH_DIPTHONG_RE.search(word)
+        if match is not None:
+            word = match.group(1) + '-' + match.group(2)
+    return word
+
+
+def syllabify(word):
     """
-    Hyphenates a word.
-    :param word: The word to be hyphenated.
+    Syllabifies a word.
+    :param word: The word to be syllabified.
     :return: Hyphenation.
```

Comment:
Return list of syllables and exceptions where appropriate, doesn't it?

Reply:
Yep, updated